calcom/cal.com
# fix: password validation

## What does this PR do?
Added Password Validation in Signup

Fixes #9215

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
- Go to https://app.cal.com/signup
- Try invalid password and you would see errors
- Correct password would let you signup
